hack/install-agent-rpm-ostree.sh.template

```diff
@@ -13,13 +13,15 @@ OPTIONS:
            kubectl port-forward service/flotta-operator-controller-manager -n flotta 8043 --address 0.0.0.0
```

Comment:
change 8043 by <FLOTTA_PORT> or something like that to emphasis that the provided port should be used. Or add ie: before the command to be clear that it's a simple example using default values.
Also, change [this](https://github.com/project-flotta/flotta-operator/blob/feb03938c5fd901b2db04904d01375735d4e3919/hack/install-agent-rpm-ostree.sh.template#L10) that describe -i option